Gems/Atom/RPI/Code/Source/RPI.Public/Scene.cpp

```diff
@@ -448,6 +449,7 @@ namespace AZ
         {
             AZ_PROFILE_SCOPE(RPI, "Scene: Simulate");
 
+            m_prevSimulationTime = m_simulationTime;
```

Comment:
The m_simulationTime doesn't have default value. So this may end up with random value in the first frame.